ministryofjustice/correspondence_tool_staff
# CT-3166 specify contact name

## Description
This PR adds support for specifying contact names for the ICO and Litigation complaint types, as well as validations and the conditional display of the values on the case details page.

## Self-review checklist


### Screenshots

![image](https://user-images.githubusercontent.com/1161161/102775818-84a43f00-4385-11eb-892a-a2c5a2bd42a5.png)
<img width="584" alt="image" src="https://user-images.githubusercontent.com/1161161/102775830-8d951080-4385-11eb-9cd0-47ad8c2309de.png">
<img width="558" alt="image" src="https://user-images.githubusercontent.com/1161161/102775851-9554b500-4385-11eb-82e4-ae0e0aa25d45.png">
<img width="655" alt="image" src="https://user-images.githubusercontent.com/1161161/102775877-a1407700-4385-11eb-905e-adc3702f722e.png">

<img width="839" alt="image" src="https://user-images.githubusercontent.com/1161161/102775976-d8af2380-4385-11eb-89ed-1d16f421b8e1.png">


### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-3166

### Deployment
n/a

### Manual testing instructions
As Branston Responder, create a complaint case
Specify ICO - verify that name, reference and a contact means (either email or phone) are mandatory
Verify the details display in the relevant section of the case details page
Same for Litigation
